doc/introduction/templates.rst

```diff
@@ -74,8 +80,51 @@ functions that act on templates, such as :func:`~.pennylane.inv`:
         return qml.expval(qml.PauliZ(0) @ qml.PauliZ(1))
 
 
-See below for details on built-in templates provided by PennyLane.
+The following is a gallery of built-in templates provided by PennyLane.
+
+
+.. _intro_ref_temp_emb:
+
+Embedding templates
+-------------------
+
+Embeddings encode input features into the quantum state of the circuit.
+Hence, they take a feature vector as an argument. Embeddings can also depend on
+trainable parameters, and they may consist of repeated layers.
+
+.. customgalleryitem::
+    :tooltip: AmplitudeEmbedding template
+    :description: :doc:`AmplitudeEmbedding</code/api/pennylane.templates.embeddings.AmplitudeEmbedding>`
+    :figure: ../_static/templates/embeddings/amplitude.png
```

Comment:
All the thumbnails look great, but amplitude embedding is a bit vague (just a white square), is there a way of displaying slightly more info? Not a major comment though.

Reply:
I thought it forth and back, and AmplitudeEmbedding is not defined by a circuit, and could in future actually depend on different execution methods depending on the device. So for all purposes it is here a black box...